pallets/maintenance-mode/src/lib.rs

```diff
@@ -181,6 +181,24 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> Pallet<T> {
+		/// Internal function to force the chain into maintenance mode without an origin check.
+		pub fn force_enter_maintenance_mode() {
```

Comment:
Calling it force is probably not adequate, lets follow the usual convention of calling no-dispatchable pallet methods with `do_*`.
```suggestion
		pub fn do_enter_maintenance_mode() {
```

Reply:
Addressed in https://github.com/Moonsong-Labs/moonkit/pull/66/commits/540018227400e9364c80d32d1f12f2fdac66891b